lib/ama/styles/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module AMA
   module Styles
-    VERSION = '2.4.2'
+    VERSION = '2.4.3'
```

Comment:
Since you are adding functionality and not fixing a bug, this should be. `2.5.0` .  (`majorVersion`.`minorVersion`.`patch`)

https://semver.org/